lib/ansible/executor/playbook_executor.py

```diff
@@ -192,7 +193,7 @@ def run(self):
                         retries = sorted(retries)
                         if len(retries) > 0:
                             if C.RETRY_FILES_SAVE_PATH:
-                                basedir = C.shell_expand(C.RETRY_FILES_SAVE_PATH)
+                                basedir = shell_expand(C.RETRY_FILES_SAVE_PATH, C.CONFIG_FILE)
```

Comment:
I'm also concerned that we are needing to change function signatures, to allow moving the code out, for what seems to be little benefit